### PR TITLE
Add pep8 naming check for python script

### PR DESCRIPTION
**Pull request by hanhsuan**
Following the discussion in sprint, adding the pep8 naming check in github action. If any naming changed in the PR wouldn't fit the original meaning, please let me know.

### CODE REVIEW

File: API/Jira/scenarios/pc/pc.py [hunk 1]
```diff
@@ -487,10 +487,10 @@ def _create_rts_task(self, story_task={}):
                 target_issues=[{'key': story_task['key']}]) \
                 if story_task['key'] else {}
 
-            issueUpdates.append({'fields': fields, 'update': update_link})
+            issue_updates.append({'fields': fields, 'update': update_link})
 
         response = self.jira_api.create_issues(
-            payload={'issueUpdates': issueUpdates})
+            payload={'issue_updates': issue_updates})
```
Comment (stanley31huang) [inline API/Jira/scenarios/pc/pc.py:493]: @baconYao for the key pattern in playload, it should stays in the original style?

Comment (baconYao) [reply API/Jira/scenarios/pc/pc.py:493]: @hanhsuan Please help to fix it as `payload={'issue_updates': issue_updates})` as we discussed this afternoon.
I've tried it and everything works well.

Thanks!

Comment (hanhsuan) [reply API/Jira/scenarios/pc/pc.py]: Fixed it.

**Review by baconYao — APPROVED** [review]
LGTM

**Pull request merged**